Shopping Cart.html

```diff
@@ -5,8 +5,27 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Shoe Store</title>
+    <link rel="stylesheet" href="src/Shopping Cart.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="Shopping Cart.css">
```
You should use relative paths for your local files, such as `<link rel="stylesheet" href="Shopping Cart.css">` instead of  `<link rel="stylesheet" href="src/Shopping Cart.css">`.This will make your website work on any server or folder structure.

Reply:
Noted bro.